stock_request/views/stock_request_order_views.xml

```diff
@@ -23,6 +23,7 @@
                         string="Confirm"
                         type="object"
                         attrs="{'invisible': [('state', 'not in', ['draft'])]}"
+                        groups="stock_request.group_stock_request_manager"
```

Comment:
Why changing this behaviour ?

Reply:
@victoralmau Stock request users should have access to confirmations. If you want to restrict, please either do it on your side or add an option to configure this behaviour.

Reply:
Perhaps another possible approach would be to add a setting to define who can use confirm button (user or manager). What do you think about this?

Reply:
Yes and protect the action method through that